src/current/v24.3/cockroach-cert.md

```diff
@@ -169,6 +169,14 @@ Flag | Description
 
 {% include {{ page.version.version }}/misc/logging-defaults.md %}
 
+### `create-client`
+
+The `cockroach cert create-client` subcommand supports the following flags:
+
+Flag | Description
+-----|-----------
+`--tenant-name-scope` | {% include_cached new-in.html version="v24.3" %} Allows you to use the tenant name when creating a client certificate and key pair, in addition to the tenant ID.
```

Comment:
The 'new in' is 'breaking' the table. Renders 'fine', just not as expected—with everything after 'new in' showing as a new paragraph after the table. Check out the preview. Can you think of any good workaround?

Being rendered like this. I assume the 'span' tag from the include is getting in the way, but I'm not sure. Maybe converting this to an HTML table is the simplest solution for now.

```
<table><thead>
<tr>
<th>Flag</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td><code>--tenant-name-scope</code></td>
<td><span class="version-tag">New in v24.3:</span></td>
</tr>
</tbody></table>

<p>Allows you to use the tenant name when creating a client certificate and key pair, in addition to the tenant ID.</p>
```

Reply:
Let's see if this does it.